src/freertos/platforms/posix/platform_freertos_hooks.c

```diff
@@ -10,3 +12,58 @@ void vApplicationIdleHook(void) {
 
     nanosleep((const struct timespec[]){{sleepSec, sleepNano}}, NULL);
 }
+
+#if configSUPPORT_STATIC_ALLOCATION == 1
+
+/* configSUPPORT_STATIC_ALLOCATION is set to 1, so the application must provide an
+implementation of vApplicationGetIdleTaskMemory() to provide the memory that is
+used by the Idle task. */
+void vApplicationGetIdleTaskMemory(StaticTask_t** ppxIdleTaskTCBBuffer,
```

Comment:
Could you just put a link for references?

Reply:
And where is this implemented for the stm32?

Reply:
From what I can tell, it's implemented by CMSIS-RTOS (which is included in global sources)